lib/src/utils/helper.dart

```diff
@@ -1,3 +1,5 @@
+import 'exceptions/blockchain_exception.dart';
+import 'exceptions/wallet_exception.dart';
 
 const MIN_WORD_COUNT= 6;
 const MAX_WORD_COUNT = 24;
```

Comment:
I think these `MIN_ENTROPY_BIT` and `MAX_ENTROPY_BIT` and `isBadEntropyBitCount()` have been removed because I can't find usage of them anywhere, so they could be removed here as well.

Reply:
Yeah these are not required anymore. Will delete.